src/particles/PhysicsParticles.hpp

```diff
@@ -469,13 +469,11 @@ template <typename problem_t> class PhysicsParticleRegister
 	}
 
 	// Update velocities of all massive particles
-	void kickParticlesAllLevels(amrex::Real dt, amrex::Vector<amrex::MultiFab> &acceleration)
+	void kickParticlesAtLevel(amrex::Real dt, amrex::MultiFab &acceleration, int lev)
```

Comment:
The particles infrastructure currently won't work with AMR subcycling. We need virtual and ghost particles for that (for details, see: https://github.com/quokka-astro/quokka/issues/332).